sfcasts/dashboard.md

```diff
@@ -1,95 +1,61 @@
+
+I'll stick with `symfony console` throughout this tutorial. So say:
+
+```terminal
+symfony console make:admin:dashboard
+```
+
+We'll call it "DashboardController."
+
+Sounds good! To generate it, say `src/Controller/admin` and... done! This creates one new file for us: `src/Controller/Admin/DashboardController.php`.
+
+Let's go check it out!
+
+Ooh... Let me see... There's not a lot here, but one thing you might notice is that it has a route for `/admin`. So now if I go over here to `/admin`... we do hit the admin dashboard!
+
+Okay, so there's a few things to notice here. First is that we do have a `/admin` route. This isn't fancy, and there's really nothing "EasyAdmin" about it. This is just how we create routes in Symfony. This is a PHP 8 attribute route, which you may or may not be familiar with. I've typically used annotations until now, but now that I'm using PHP 8, I'll be using attributes instead of annotations. Don't worry, though! They work exactly the same. If you're still using PHP 7, then you can use annotations just fine.
+
+The `DashboardController` is just a normal controller, but it extends `AbstractDashboardController`. I'll hold "Cmd" and open that. 
+
+This implements `DashboardControllerInterface`. So this is a controller, but by implementing the interface, it gives it special superpowers. There are a number of methods inside of here that we can override to configure what our dashboard looks like. We'll be doing that little by little throughout this tutorial. 
+
+Because this is just a normal route in controller, there's really nothing special about it. It's not automatically secured and there's no security being applied to this. I mean, check it out: I'm not even logged in right now, and I am successfully on my admin dashboard.
+
+So let's secure it! I'll also do this with annotations. I already have SensioFrameworkExtraBundle installed. So I can say `#[IsGranted()]` and hit "tab" to auto complete that. Now we'll look for `ROLE_ADMIN`. That's kind of my base admin role, so all admin users will have that role. Now when I refresh... Beautiful! We bounced back over to the login page!
+
+So to log in, if you open `src/DataFixtures/AppFixtures`, I have a bunch of dummy users in the database. There is a super admin, a normal admin, and then somebody known as a moderator. We'll talk more about those later, when we get deeper into how to secure different parts of your admin with different roles.
+
+I'll log in with "admin@example.com"... password "admin password"...
+
+And... beautiful! Now we're back to our dashboard!
+
+Of course, if you want to, instead of securing `IsGranted` with the PHP attribute, you can use `$this->denyAccessUnlessGranted()`. You can also go to `config/packages/security.yaml` and, down at the bottom, you can always just use access control for this - whichever one you like better.
```

Comment:
? Of course, instead of securing with the PHP attribute, you can use...

This is a spot where I totally tripped over myself and I say half a sentence that I don't want to. If you listen to the audio, it's (hopefully) obvious what I intended to say vs what were words that I regretted saying after I started ;)

Reply:
> down at the bottom, use `access_control` for this - whichever one you like better

2 minor things:

* I had `can` twice in that sentence, so removing it here
* "access control" is `access_control`: a technical thing that you can see in the `security.yaml` file on the video